src/customHooks/useDisruptionAffectedItems.js

```diff
@@ -47,20 +58,34 @@ const useDisruptionAffectedItems = (disruption) => {
       </div>
       <div className="wmnds-col-1">
         {/* Affected Services / Bus */}
-        {disruption.servicesAffected &&
-          disruption.mode === 'bus' &&
-          disruption.servicesAffected
-            .sort((a, b) => a.serviceNumber.replace(/\D/g, '') - b.serviceNumber.replace(/\D/g, ''))
-            .map((affected) => (
-              <FavBtn
-                id={affected.id}
-                severity={disruption.disruptionSeverity}
-                text={affected.serviceNumber}
-                title={`${affected.routeDescriptions[0].description} (${affected.operatorName})`}
-                mode={disruption.mode}
-                key={affected.id}
+        {disruption.servicesAffected && disruption.mode === 'bus' && (
+          <>
+            {disruption.servicesAffected
+              .sort(
+                (a, b) => a.serviceNumber.replace(/\D/g, '') - b.serviceNumber.replace(/\D/g, '')
+              )
+              .slice(0, sliceUpper)
+              .map((affected) => (
+                <FavBtn
+                  id={affected.id}
+                  severity={disruption.disruptionSeverity}
+                  text={affected.serviceNumber}
+                  title={`${affected.routeDescriptions[0].description} (${affected.operatorName})`}
+                  mode={disruption.mode}
+                  key={affected.id}
+                />
+              ))}
+            {disruption.servicesAffected.length > maxShownBeforeHiding && (
+              <ToggleMoreAffectedItems
+                handleClick={toggleExpanded}
+                id={`toggleMoreAffectedItems_${disruption.id}`}
+                isExpanded={isExpanded}
+                amountHidden={disruption.servicesAffected.length - maxShownBeforeHiding}
+                serviceText={whatIsAffected}
```

Comment:
This is always plural i.e. "services", if the remaining service(s) behind the button is only 1 then it should say "Show 1 more service".

Do a check on the disruption.servicesAffected.length - maxShownBeforeHiding or amountHidden var and if it equals 1 then it should be singular (service) else it should be plural.

Example of the bug: https://deploy-preview-429--wmn-disruptions.netlify.app/?isMapVisible=false&mode=bus&query=997&selectedItem=4738

![image](https://user-images.githubusercontent.com/13015201/102195109-0f80c780-3eb6-11eb-80ff-d5a610b1c7ce.png)

Reply:
Cool. Do you also think it's worth changing the amount shown to 6 so that it all fits on one line?